library/unwind/src/libunwind.rs

```diff
@@ -37,7 +37,7 @@ pub const unwinder_private_data_size: usize = 20;
 pub const unwinder_private_data_size: usize = 5;
 
 #[cfg(all(target_arch = "aarch64", target_pointer_width = "64"))]
-pub const unwinder_private_data_size: usize = 2;
+pub const unwinder_private_data_size: usize = 6;
```

Comment:
Should this change be limited to just `windows-gnullvm`?

Reply:
Dunno, in the past similar change has been done globally: https://github.com/rust-lang/rust/pull/16261/commits/5a24ee8a9ef1bb3bf250ce39902cf8dfca1a4dea#diff-698c629995c6d73852b1f89091513361bf9cd26d6d3e54e4937a2fcb66075e37R63
AFAIK the only downside is wasting few bytes of memory for others but I can add cfg of you want.

Reply:
It's easier for me to be sure this won't cause issues on other targets -- It's not clear to me if this is declaring our own data layout, or needs to match something else.

Reply:
We are matching libgcc/libunwind layout here on a way to allocate memory sufficient for all users (so overallocating/wasting memory for some targets).
I'll move it to separate cfg. Should I do the same for x86_64 Windows (basically reverting generic x86_64 size to 2 that was before https://github.com/rust-lang/rust/commit/5a24ee8a9ef1bb3bf250ce39902cf8dfca1a4dea#diff-698c629995c6d73852b1f89091513361bf9cd26d6d3e54e4937a2fcb66075e37R63 and moving it to item common with AArch64 Windows)?

Reply:
In a different PR, yeah

Reply:
Ok, this PR should be ready to go then.